app/helpers/validators_helper.rb

```diff
@@ -43,7 +42,12 @@ def current_software_version(batch, network)
   def skipped_vote_percent(validator, batch, skipped_vote_percent_best)
```

Comment:
Wouldn't it be better to have it in the model?

Reply:
It's not about my changes.